README.md

```diff
@@ -43,3 +43,42 @@ Using the command console, enter the following command:
 ```git clone <link to your repository>```
 
 And Voilá! The repository is cloned to your computer!
+
+
+
+
+How to Submit a Pull Request
+
+To submit a Pull Request, follow these steps:
+
+
+•	Find a repository on Github
```

Comment:
These instructions should be specific to this project, and not GitHub in general. Please remove this first bullet point.

Reply:
I have removed this bullet point